website/content/docs/commands/operator/debug.mdx

```diff
@@ -54,6 +54,14 @@ true.
 - `-interval=2m`: The interval between snapshots of the Nomad state.
   If unspecified, only one snapshot is captured.
 
+-targets=<metrics,-metrics,...,all,none>
+  Comma separated list of targets to capture.  Defaults to "all".
+  "-<target>" disables capture for that target.
+
+-interval-targets=<metrics,-metrics,...,all,none>
+  Comma separated list of targets to capture during each interval.  Defaults
+  to "all".  "-<target>" disables capture for that target.
```

Comment:
```suggestion
  to "all".  Note that prefixing a target name with the `-` operator (e.g. "-metrics") disables capture for that target.
```

Reply:
Updated these both on the website docs and in the CLI help.